vendor-extra/JatsContentBundle/tests/EventListener/BuildView/ExtLinkListenerTest.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\JatsContentBundle\EventListener\BuildView;
+
+use Libero\JatsContentBundle\EventListener\BuildView\ExtLinkListener;
+use Libero\ViewsBundle\Event\BuildViewEvent;
+use Libero\ViewsBundle\Event\ChooseTemplateEvent;
+use Libero\ViewsBundle\Views\TemplateView;
+use PHPUnit\Framework\TestCase;
+use tests\Libero\LiberoPageBundle\ViewConvertingTestCase;
+use tests\Libero\LiberoPageBundle\XmlTestCase;
+
+final class ExtLinkListenerTest extends TestCase
+{
+    use ViewConvertingTestCase;
+    use XmlTestCase;
+
+    /**
+     * @test
+     * @dataProvider templateChoiceProvider
+     */
+    public function it_can_choose_a_template(string $xml, ?string $expected) : void
+    {
+        $listener = new ExtLinkListener($this->createFailingConverter());
+
+        $element = $this->loadElement($xml);
+
+        $event = new ChooseTemplateEvent($element);
+        $listener->onChooseTemplate($event);
+
+        $this->assertSame($expected, $event->getTemplate());
+    }
+
+    public function templateChoiceProvider() : iterable
+    {
+        yield 'ext-link element' => [
+            <<<XML
+<ext-link xlink:href="bar" xmlns="http://jats.nlm.nih.gov" xmlns:xlink="http://www.w3.org/1999/xlink">
+    foo
```

Comment:
slightly confusing that the positive example is a relative link (`bar`) which will then be ignored  by https://github.com/libero/browser/pull/104/files#diff-663e4050bbaaab66f585fa7bf668af63R35

Reply:
Ah yeah, they should be absolute unless deliberately relative.